src/java.desktop/windows/native/libawt/windows/awt_Win32GraphicsDevice.cpp

```diff
@@ -179,16 +179,7 @@ void AwtWin32GraphicsDevice::Initialize()
     }
     gpBitmapInfo->bmiHeader.biBitCount = 0;
     HDC hBMDC = this->GetDC();
-    VERIFY(hBMDC != NULL);
```

Comment:
You may want to leave `VERIFY(hBMDC != NULL)`… just in case.

Reply:
I you don't add the assertions for `hBMDC` and `hBM`, you should revert all the changes to `awt_Win32GraphicsDevice.cpp`.

At the moment, two lines are removed from `initScreens`.

Reply:
The two lines removed from initScreens are in `awt_Win32GraphicsEnv.cpp` 😉  That's unrelated cleanup I guess. But I bring back the VERIFY code which seems to make sense and would not alter the behavior in the non-debug case.

Reply:
You're right! `awt_Win32GraphicsDevice.cpp` is not part of the PR at this very moment.